packages/mobile/src/app/App.tsx

```diff
@@ -66,6 +67,23 @@ export class App extends React.Component<Props> {
   async componentDidMount() {
     await ValoraAnalytics.init()
 
+    // Handles opening Clevertap deeplinks when app is closed / in background
+    CleverTap.getInitialUrl(async (err: any, url: any) => {
+      if (err) {
+        Logger.debug('App', err)
+      } else if (url) {
+        await this.handleOpenURL({ url }, true)
+      }
+    })
+
+    // Handles opening Clevertap deeplinks when app is open
+    CleverTap.addListener('CleverTapPushNotificationClicked', async (event: any) => {
+      if (event.customExtras['wzrk_dl']) {
```

Comment:
This looks quite strange to me 😅  What is `wzrk_dl`?

Reply:
`wzrk_dl` appears to be the deeplink field provided by CleverTap.

Reply:
Wow, that's a weird choice from them. Is this documented or did you figure it out? I just wonder if it's going to change across different installs or something :/ 

In any case I think we should probably move it to a constant, it looks a bit magic-y like this

Reply:
I haven't seen it change across installs. Their documentation calls it out [here](https://docs.clevertap.com/docs/faq#q-why-am-i-receiving-blank-push-notifications).

Reply:
Oh, I guess `dl` stands for `deep link`, I wonder what `wzrk` stands for 😅

Reply:
I have no idea what `wzrk` stands for, but they use it a bunch of places across different SDKs. 🤷

Reply:
> In any case I think we should probably move it to a constant

Something like:

```JavaScript
const url = event?.customExtras['wzrk_dl']
if (url) {
  await this.handleOpenURL({ url }, true)
}
```

Reply:
Yes this seems good.